catalyst/utils/image.py

```diff
@@ -2,6 +2,7 @@
 import os
 import numpy as np
 import imageio
+import jpeg4py as jpeg
```

Comment:
Import should be moved to lines 35-36 (inside condition) because now it's optional and should be removed from requirements.